assets/javascripts/update_issues_list.js

```diff
@@ -22,7 +22,7 @@ export default class UpdateIssuesList {
 
     static removeIssue(element) {
         const parent = element.parent();
-        parent.remove();
+        parent.parent().remove();
```

Comment:
JS, am I right?

The clean way would be to create a reference in some other way but I understand if you say: nope

Reply:
While I'm here. Three of the static functions above can just be constants, right?
Or you could at least define them as arrow functions:
```js
static issueDeletionButton = () => 'data-update-issue-deletion-action'; // Implicit return in js (only works for one liners)
```